app/e2e/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Is this the recommended config from playwright or one autogenerated from tsc --init?

Reply:
No I had just copied that over from the client and edited it. Should I delete it and recreate with tsc --init?

Reply:
Based on the wording here, it seems that typescript should work with playwright by default without a tsconfig.json

See https://playwright.dev/docs/test-typescript#manually-compile-tests-with-typescript
>Sometimes, Playwright Test will not be able to transform your TypeScript code correctly, for example when you are using experimental or very recent features of TypeScript, usually configured in tsconfig.json. **In this case, you can perform your own TypeScript compilation before sending the tests to Playwright. First add a tsconfig.json file inside the tests directory:**

emphasis mine

So I think it should be removed?

Reply:
I tried doing it that way but I get errors when trying to run the tests after they are compiled.
![Screenshot from 2022-06-16 17-43-06](https://user-images.githubusercontent.com/14946745/174200319-f09da27c-bbc0-4f27-986c-c8b9da3fadb0.png)

Reply:
I'm sure there is a way to fix this, but I'm not sure how much time I should spend on this when it is working the current way. If you want to work on fixing it together sometime though let me know.

Reply:
We can work on this in another PR. I'll go over the project in general and do some house keeping.